src/github/middleware.ts

```diff
@@ -142,7 +142,7 @@ export default (
 					},
 					`Timing out at after ${context.timedout}ms`
 				);
-				return;
+				continue;
```

Comment:
Actually, this return was plain wrong. After a timeout we should at least try to send the data to the next Jira. But it was always wrong...